packages/drawer/src/views/Drawer.tsx

```diff
@@ -214,11 +219,12 @@ export default class DrawerView extends React.PureComponent<Props> {
   private velocityX = new Value<number>(0);
   private gestureX = new Value<number>(0);
   private offsetX = new Value<number>(0);
-  private position = new Value<number>(0);
+  private position = new Value<number>(this.bigScreenSidebar ? 1 : 0);
```

Comment:
Probably don't need to adjust `position`. We could probably hardcode the value in render if we need it